diagnostic.md

```diff
@@ -58,13 +62,14 @@ Suppose you want to define a mixin named `row` stored in `./row.scss`. Write the
 code to import the mixin definition in the current module.
 
 ```scss
-// your response here
+// @import "row";
+# row.scss
```

Comment:
WHy this second comment? the first is correct as long as you change the double quotes to single